README.md

```diff
@@ -1,2 +1,3 @@
 # github-reusable-workflows
+
```

Comment:
add an example of calling it?

Reply:
I think it would suffice to edit https://github.com/jenkinsci/.github/blob/1292a5224fbc08abedc30408c14f71e804c95bbb/workflow-templates/cd.yaml#L11-L61